kotsadm/web/src/components/apps/Dashboard.jsx

```diff
@@ -124,13 +125,18 @@ class Dashboard extends Component {
       }
     }).then(async (res) => {
       const body = await res.json();
-      if (body === null) {
-        this.setState({ appLicense: {} });
+      if(!res.ok) {
+        this.setState({ getingAppLicenseErrMsg: body.error });
```

Comment:
i don't think `body.error` is defined here

Reply:
I think it may be. This is one Jelena had done and I believe this screenshot is the test of it. https://user-images.githubusercontent.com/25138806/89321587-06463b00-d638-11ea-8e75-b741d7ac12be.png

Reply:
i don't know how Jelena tested this, but how can there be a json parsed `body` if the response is not `ok`? the `error` here is assumed to be returned from the api, which cannot be the case here, because the requested failed with a network error.

Reply:
@GraysonNull i think the screenshot is showing this error https://github.com/replicatedhq/kots/pull/902/files/a8d84ede7a4850a042470ea3226ff71b95221075#diff-174a420164b7a9cd5f4d451644e971b3R139